src/m3ninx/index/segment/builder/builder_test.go

```diff
@@ -78,6 +78,7 @@ var (
 
 func TestBuilderFields(t *testing.T) {
 	builder, err := NewBuilderFromDocuments(testOptions)
+	defer builder.Close()
```

Comment:
Since this is a test, instead of deferring close here, maybe add `require.NoError(t, builder.Close())` at the end of the test?